samples/scenarios/extract_only/sqlite_extract/dialect.xml

```diff
@@ -1,5 +1,1319 @@
+      <formula part='hour'>(CAST(%1 AS DATE) + EXTRACT(HOUR FROM %1) * INTERVAL &apos;1 HOUR&apos;)</formula>
+      <formula part='minute'>((CAST(%1 AS DATE) + EXTRACT(HOUR FROM %1) * INTERVAL &apos;1 HOUR&apos;) + EXTRACT(MINUTE FROM %1) * INTERVAL &apos;1 MINUTE&apos;)</formula>
+      <formula part='second'>(((CAST(%1 AS DATE) + EXTRACT(HOUR FROM %1) * INTERVAL &apos;1 HOUR&apos;) + EXTRACT(MINUTE FROM %1) * INTERVAL &apos;1 MINUTE&apos;) + EXTRACT(SECOND FROM %1) * INTERVAL &apos;1 SECOND&apos;)</formula>
+      <argument type='localstr' />
+      <argument type='datetime' />
+      <argument type='localstr' />
+    </date-function>
+  </function-map>
+  <supported-aggregations>
+    <aggregation value='AGG_SUM' />
+    <aggregation value='AGG_AVG' />
+    <aggregation value='AGG_MIN' />
+    <aggregation value='AGG_MAX' />
+    <aggregation value='AGG_STDEV' />
+    <aggregation value='AGG_STDEVP' />
+    <aggregation value='AGG_VAR' />
+    <aggregation value='AGG_VARP' />
+    <aggregation value='AGG_COUNT' />
+    <aggregation value='AGG_COUNTD' />
+    <aggregation value='AGG_YEAR' />
+    <aggregation value='AGG_QTR' />
+    <aggregation value='AGG_MONTH' />
+    <aggregation value='AGG_DAY' />
+    <aggregation value='AGG_HOUR' />
+    <aggregation value='AGG_MINUTE' />
+    <aggregation value='AGG_SECOND' />
+    <aggregation value='AGG_WEEK' />
+    <aggregation value='AGG_WEEKDAY' />
+    <aggregation value='AGG_MONTHYEAR' />
+    <aggregation value='AGG_MDY' />
+    <aggregation value='TRUNC_YEAR' />
+    <aggregation value='TRUNC_QTR' />
+    <aggregation value='TRUNC_MONTH' />
+    <aggregation value='TRUNC_WEEK' />
+    <aggregation value='TRUNC_DAY' />
+    <aggregation value='TRUNC_HOUR' />
+    <aggregation value='TRUNC_MINUTE' />
+    <aggregation value='TRUNC_SECOND' />
+  </supported-aggregations>
+  <sql-format>
+    <date-literal-escape value='Standard' />
+    <format-bool-as-value value='TrueFirst' />
+    <format-column-definition>
+      <local-type name='int'>
+        <remote-type name='I2' value='SMALLINT' />
+        <remote-type name='I4' value='INTEGER' />
+        <remote-type name='default' value='BIGINT' />
+      </local-type>
+      <local-type name='real'>
+        <remote-type name='R4' value='FLOAT(24)' />
+        <remote-type name='default' value='FLOAT(53)' />
+      </local-type>
+      <local-type name='str'>
+        <remote-type name='default' value='VARCHAR(%w)' />
+      </local-type>
+      <local-type name='datetime'>
+        <remote-type name='TIME' value='TIME' />
+        <remote-type name='DATE' value='DATE' />
+        <remote-type name='default' value='TIMESTAMP' />
+      </local-type>
+      <local-type name='date'>
+        <remote-type name='default' value='DATE' />
+      </local-type>
+      <local-type name='bool'>
+        <remote-type name='default' value='BOOLEAN' />
+      </local-type>
+      <local-type name='spatial'>
```

Comment:
I don't know, but I doubt sqlite supports spatial type.

Reply:
[Googled some info about this](https://pro.arcgis.com/en/pro-app/help/data/databases/overview-database-spatial-types.htm), it looks like sqlite supports spatial type, I originally added spatial type was because [CDataExtractDialect](https://opengrok.prod.tableautools.com/source/xref/teams_near/modules/connectors/tabmixins/main/db/CDataExtractDialect.cpp#123) has it.
so I removed spatial type, and after removing it, format-column-definition is same with base implementation, so deleted the whole format-column-definition.